lib/srv/db/sqlserver/protocol/constants.go

```diff
@@ -32,8 +32,8 @@ const (
 	// packetHeaderSize is the size of the protocol packet header.
 	packetHeaderSize = 8
 
-	// packetStatusLast indicates that the packet is the last in the request.
-	packetStatusLast uint8 = 0x01
+	// PacketStatusLast indicates that the packet is the last in the request.
```

Comment:
Would you mind adding the docs link?
https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-tds/ce398f9a-7d47-4ede-8f36-9dd6fc21ca43